iaf-commons/pom.xml

```diff
@@ -86,5 +86,16 @@
 			<artifactId>junit-jupiter</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.xmlbeans</groupId>
+			<artifactId>xmlbeans</artifactId>
+			<version>5.1.1</version>
+			<exclusions>
+				<exclusion><!-- Java 8 scoped profile -->
+					<groupId>xml-apis</groupId>
+					<artifactId>xml-apis</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
```

Comment:
Wel heel erg stom eigenlijk dat we een hele dependency hebben voor een enkele methode... Misschien dat je kan kijken wat de input van die methode is, daar een hele rits aan tests voor maakt, en dan deze dependency kan vervangen door native code.

Reply:
Dit mag overigens in een los pr! Is iig wel handiger, geisoleerd ding etc

Reply:
Ga ik doen, dit is puur voor het parsen van die Date natuurlijk. Is er nog een bepaald format dat wordt aangehouden, is het altijd datum en tijd of kan dit verschillen, dus alleen datum zijn?

Reply:
Beide mag, het heeft geen haast, maar het moet wel gebeuren.